### ISSUE TITLE
Feature: Feeds page UX improvements

### ISSUE THREAD

**Issue by OgDev-01** [open-sauced/insights#1451]
### Type of feature

🍕 Feature

### Current behavior

![new update to feeds page illustration](https://github.com/open-sauced/insights/assets/62995161/aa0c5fba-677c-4f8c-83ab-01cfeff56250)

### Concepts
Similar to [linkedin](https://www.linkedin.com/feed) we can improve our UX(user experience) by making some areas sticky for quick accessibility. The above screenshot point out areas that should be affected 👌

### Suggested solution

The Nav should be fixed... and every other area alighted should be sticky in relation to the position of the nav with like about 20-20 pixels gap from the top.

### Note
The only area to keep scrolling is the highlights section

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

**Comment by a0m0rajab** [comment]
Current behavior:
 
https://github.com/open-sauced/insights/assets/18273833/53ebf1a7-0d0f-4747-b157-450c64a4657f

**Comment by a0m0rajab** [comment]
You are welcome.

**Comment by takanome-dev** [comment]
duplicate of https://github.com/open-sauced/insights/issues/1353

**Comment by bdougie** [comment]
moved to papercuts, ups for grabs.

**Comment by dev-phantom** [comment]
please can i take on this issue?
